tests/data/test_custom_datasets.py

```
@@ -715,3 +716,42 @@ def test_single_instance_dataset(minimal_instance, tmp_path):
+        confmap_head_config=confmap_head,
+        labels=labels,
+        apply_aug=base_singleinstance_data_config.use_augmentations_train,
+        np_chunks=False,
+    )
+
+    assert len(list(iter(dataset))) == 1
+
+    dl = iter(CyclerDataLoader(dataset, batch_size=1, num_workers=0))
+
+    for _ in range(5):
+        ex = next(dl)
```

Ruff (0.8.2)

Local variable `ex` is assigned to but never used

Remove assignment to unused variable `ex`

(F841)